rust/workspace-crate/{{ cookiecutter.crate_sub_dir }}/Cargo.toml

```
@@ -1,16 +1,20 @@
 [package]
 name = "{{ cookiecutter.crate_name }}"
 version = "{{ cookiecutter.version }}"
-authors = ["MobileCoin"]
+authors = { workspace = true }
 # See https://crates.io/category_slugs for valid categories
 categories = ["{{ cookiecutter.crate_categories.split()|join('", "') }}"]
 description = "{{ cookiecutter.crate_description }}"
-edition = "2021"
+edition = { workspace = true }
 # See https://crates.io/keywords for the common keywords
 keywords = ["{{ cookiecutter.crate_keywords.split()|join('", "') }}"]
-license = "Apache-2.0"
+{%- if cookiecutter.license == "Same as Workspace" -%}
+license = { workspace = true }
+{%- else -%}
+license = "{{ cookiecutter.license }}"
+{%- endif -%}
```

:x: Sorry somehow I failed to actually add this comment in the previous version
Anyway the trailing `-` results in invalid toml when rendered.
```suggestion
{%- if cookiecutter.license == "Same as Workspace" %}
license = { workspace = true }
{%- else %}
license = "{{ cookiecutter.license }}"
{%- endif %}
```

---

it looks like this fix got lost along the way